airmozilla/manage/static/manage/js/survey-edit.js

```diff
@@ -31,19 +31,28 @@ $(function() {
     // hijack the ability to save/edit
     $('form.edit').submit(function() {
         var form = $(this);
+        $(form.children('button')).text('Saving..');
```

Comment:
Related but unrelated. You're here doing a `$(already_a_jquery_object)`. The code `form.children('button')` already is a "jQuery array" thing.